templates/zabbix_agentd.conf.erb

```diff
@@ -216,38 +205,32 @@ MaxLinesPerSecond=<%= @maxlinespersecond %>
         <%- end -%>
 <% end %>
 
-
 ### Option: Timeout
 #	Spend no more than Timeout seconds on processing
 #
 Timeout=<%= @timeout %>
 
-<% if @kernel != 'windows' %>
 ### Option: AllowRoot
 #	Allow the agent to run as 'root'. If disabled and the agent is started by 'root', the agent
 #	will try to switch to the user specified by the User configuration option instead.
 #	Has no effect if started under a regular user.
 #	0 - do not allow
 #	1 - allow
 #
-AllowRoot=<%= @allowroot %>
-<% end %>
+<% if @allowroot && @kernel != 'windows' %>AllowRoot=<%= @allowroot %><% end %>
```

Comment:
The if here was supposed to be on top to remove the comments as well btw

Same for the rest of these

Reply:
Right, but do not understand the issue of having more comments.